### DISCUSSION TITLE
Queries regarding Parameter Set Selection

### DISCUSSION THREAD

**Discussion by ghost** [pybamm-team/PyBaMM#2257] [Q&A]
I have a small query related to Parameter Set selection. Normally, we select "Chen2020"(LG 21700) for Liionpacks, but if my cell configuration is 18650, which parameter set should I prefer?

**Comment by TomTranter** [comment]
You could try loading in any of them and checking the nominal capacity

**Reply by ghost** [reply]
Sir, what I understand is that when we load the "Chen 2020" parameter set, there should be some internal parameters which should be some internal parameters which behave as LG 21700 cells. As per your suggestion, if I load the "Chen 2020" parameter set and give parameters (as netlist) for 18650 cells, it doesn't make any sense. I gave the parameters and it is working but still, I'm confused that if both configurations behave similar for the "Chen 2020" parameter set, there is no valid significance of the research paper.

**Reply by brosaplanella** [reply]
From an electrochemical point of view, it will only change the capacity of the battery which you can adjust by changing the electrode area (by changing `Electrode width [m]` and/or `Electrode height [m]`). 

From a thermal point of view, it will have a bigger impact, but if you used a lumped model you only need to adjust `Cell cooling surface area [m2]` and `Cell volume [m3]`.
